production/helm/loki/CHANGELOG.md

```diff
@@ -13,6 +13,9 @@ Entries should include a reference to the pull request that introduced the chang
 
 [//]: # (<AUTOMATED_UPDATES_LOCATOR> : do not remove this line. This locator is used by the CI pipeline to automatically create a changelog entry for each new Loki release. Add other chart versions and respective changelog entries bellow this line.)
 
+## 5.41.8
```

Comment:
Can you make this 5.41.9?

Also you checked this but did not do the steps in it. For Helm chart changes bump the Helm chart version in production/helm/loki/Chart.yaml and update production/helm/loki/CHANGELOG.md and production/helm/loki/README.md. [Example PR](https://github.com/grafana/loki/commit/d10549e3ece02120974929894ee333d07755d213)

Reply:
I merged in main, which might have caused some of those changes to disappear as they are the same as main is now.

Reply:
@MichelHollands  Done